deliveries/admeta_milestone_1.md

```diff
@@ -0,0 +1,28 @@
+# Milestone Delivery :mailbox:
+
+**The [invoice form :pencil:](https://docs.google.com/forms/d/e/1FAIpQLSfmNYaoCgrxyhzgoKQ0ynQvnNRoTmgApz9NrMp-hd8mhIiO0A/viewform) has been filled out correctly for this milestone and the delivery is according to the official [milestone delivery guidelines](https://github.com/w3f/Grants-Program/blob/master/docs/milestone-deliverables-guidelines.md).**  
+
+* **Application Document:** https://github.com/w3f/Grants-Program/blob/master/applications/AdMeta.md
+* **Milestone Number:** 1
+
+
+**Deliverables**
+
+| Number | Deliverable | Link | Notes |
+| ------------- | ------------- | ------------- |------------- |
```

Comment:
In the meantime, could you please add the license in the deliverables?
```suggestion
| ------------- | ------------- | ------------- |------------- |
| 0a. | License | [LICENSE](https://github.com/AdMetaNetwork/admeta/blob/main/LICENSE) | GPLv3 |
```

Reply:
Oh sure! Thanks for the suggestion!